src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -4920,9 +4920,12 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     Leave the Clean cache file containing only the files produced in the current build.
     ============================================================
     -->
+  <PropertyGroup>
+    <IncrementalCleanDependsOn>_CleanGetCurrentAndPriorFileWrites</IncrementalCleanDependsOn>
```

Comment:
Do you think it would be better to have an if-not-defined condition here? Makes it easier to hook into.